apps/api/v2/src/ee/event-types/event-types_2024_04_15/controllers/event-types.controller.e2e-spec.ts

```diff
@@ -375,13 +405,17 @@ describe("Event types Endpoints", () => {
     it(`/GET/:id not existing`, async () => {
       await request(app.getHttpServer())
         .get(`/api/v2/event-types/1000`)
+        .set(CAL_API_VERSION_HEADER, VERSION_2024_04_15)
```

Comment:
Is it needed to add this header to all these calls since it's the default and fallback when the header isn't supplied?

Reply:
Technically no, but I added it because the modified controller now has version array `version: []` instead of string `version: string` and I just make sure that it works as expected.